docs/Developers/Git_Basics.rst

```diff
@@ -110,6 +122,15 @@ It will give you an idea about what files are currently modified.
 Adding the files and commiting
 ------------------------------
 
+First, make sure you're on the correct branch and not developing on master! If
```

Comment:
Yes, we should mention that `git checkout -b <branchname>` can be used to create a branch.  This tip should be in the Pull Request section.  i.e. 

> You've made it this far, and you're still using 'master'.  Very naughty, but here is how you can fix it before you submit a Pull Request.